app/assets/stylesheets/modern-business.scss

```diff
@@ -241,3 +247,33 @@ footer {
   font-size: 40px;
   text-align: center;
 }
+
+/* Countdown Clock */
+
+#clockdiv{
+  font-family: sans-serif;
```

Comment:
Properties should be ordered color, display, font-family, font-size, font-weight, text-align